src/ezdxf/entities/hatch.py

```diff
@@ -963,7 +963,7 @@ def export_dxf(self, tagwriter: 'TagWriter') -> None:
 
 
 def _transform_2d_ocs_vertices(ucs, vertices, elevation, extrusion) -> List[Tuple[float, float]]:
-    ocs_vertices = (Vector(x, y, elevation) for x, y in vertices)
```

Comment:
If you get en error here you are doing something wrong, HATCH only supports 2D elements, don't use 3D vertices for hatch boundary paths.